themes/default/content/docs/pulumi-cloud/access-management/saml/okta.md

```diff
@@ -22,36 +22,37 @@ This guide walks you through configuring Okta as a SAML SSO identity provider (I
 
 - [Single Sign-On](/docs/pulumi-cloud/access-management/saml/sso/)
 
-> **Note:** The screen shots below are using the Okta _Classic UI_. You can switch to it by clicking the gear
-> icon on the upper right corner of the screen.
-
 ## Creating the Okta Application
 
-The first step is to create a new Okta Application Integration. Of the various "sign on methods"
+The first step is to create a new Okta Application Integration. Of the various "sign-in methods"
 available, choose **SAML 2.0**.
 
 ![Creating an Okta Application](/images/docs/reference/service/saml-okta/create-okta-application.png)
 
 ### Configuring the Application
 
 Next you will be guided through a wizard to configure the Okta application. The first step is to
-give it a name---Pulumi Cloud for example---and an icon.
+give it a name---Pulumi Cloud for example---and an [icon](https://www.pulumi.com/brand/).
 
 ![Configuring a SAML Integration](/images/docs/reference/service/saml-okta/create-saml-integration.png)
 
 The next step is to configure the SAML application's settings.
 
-{{< saml-warning >}}
+    {{% notes type="info" %}}
+The values you need to use are dependent upon your Pulumi organization name. Be sure to replace `<orgName>` with your actual organization name.
+    {{% /notes %}}
 
 | SAML Setting | Value |
 | --------------- | ----- |
-| Single Sign-on URL | `https://api.pulumi.com/login/<acmecorp>/sso/saml/acs` |
-| Audience URI | `https://api.pulumi.com/login/<acmecorp>/sso/saml/metadata` |
-| Default Relay State | `https://api.pulumi.com/login/<acmecorp>/sso` |
-| Name ID Format | EmailAddress or Persistent |
-| App username | Email |
+| Single Sign-on URL | `https://api.pulumi.com/login/<orgName>/sso/saml/acs` |
+| Audience URI | `https://api.pulumi.com/login/<orgName>/sso/saml/metadata` |
+| Default Relay State | `https://api.pulumi.com/login/<orgName>/sso` |
+| Name ID Format | `Persistent` (`EmailAddress` will also work, but is not recommended) |
+| App username | `Email` |
 
+    {{% notes type="warning" %}}
 > **Important:** Do not change the value of the Name ID Format once your users have started using Pulumi---not even switching its value between `EmailAddress` or `Persistent`.
```

Comment:
```suggestion
> **Important:** Do not change the value of the Name ID Format once your users have started using Pulumi.
```

Might also be good to explain briefly why. If I were to read this myself, my next question would be, "What would happen if I did?"